packages/slate-plugins/src/normalizers/withNormalizeTypes.ts

```diff
@@ -44,6 +44,7 @@ export const withNormalizeTypes = ({ rules, onError }: WithNormalizeTypes) => <
         if (node) {
           if (strictType && node.type !== strictType) {
             Transforms.setNodes(editor, { type: strictType }, { at: path });
+            return;
```

Comment:
Because it's in a `forEach` scope, the return has no effect there. You can replace `forEach` by `some` and return true inside, then return outside if the result is true.